packages/conform-dom/intent.ts

```diff
@@ -142,15 +145,15 @@ export function parseIntent<Schema>(intent: string):
 }
 
 export function updateList<Schema>(
-	list: Array<Schema>,
+	list: Array<Schema | undefined>,
 	payload: ListIntentPayload<Schema>,
-): Array<Schema> {
+): Array<Schema | undefined> {
```

Comment:
this had to change, there's a possiblity that this doesn't make sense, since I believe there's some optionality happening in the ListIntentPayload type, which might indicate that it might be worth being pulled out or rearanged to make this make sense during actual runtime usage.